backend/modules/auth.ts

```diff
@@ -1,9 +1,10 @@
 import { type NextFunction, type Request, type Response } from 'express'
 
 export const checkUserAuthorization = (req: Request, res: Response, next: NextFunction): void => {
-  if (req.user === null) {
+  if (req.user === null || req.user === undefined) {
```

Comment:
This checkUserAuthorization function failed when tested with postman, but now it will work. It will throw an error when it cannot identify a user from a cookie.